Makefile

```diff
@@ -25,6 +25,9 @@ build-test:
 build-release:
 	ANOMA_DEV=false $(cargo) build --release --package namada_apps --manifest-path Cargo.toml
 
+build-debug:
+	ANOMA_DEV=false $(cargo) build --package namada_apps --manifest-path Cargo.toml
+
```

Comment:
This is useful though would separate it out from this PR, it should go towards `main` (maybe `build-apps-release` and `build-apps-debug` would make more sense as target names but it's separate from this PR)